pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -461,3 +465,12 @@ func (m *manager) updateContainerCPUSet(containerID string, cpus cpuset.CPUSet)
 			CpusetCpus: cpus.String(),
 		})
 }
+
+func (m *manager) GetCPUs(podUID, containerName string) []uint32 {
+	cpus := m.state.GetCPUSetOrDefault(string(podUID), containerName)
+	result := []uint32{}
+	for cpu := range cpus.ToSliceNoSort() {
```

Comment:
This line should be corrected to : `for _, cpu := range cpus.ToSliceNoSort() ` The current implementation results in the index values being stored in the result slice (e.g. []int32{0,1,2,3,4....} )

Reply:
exactly